src/sentry/tasks/servicehooks.py

```diff
@@ -46,7 +46,7 @@ def process_service_hook(servicehook_id, event, **kwargs):
 
     from sentry import tsdb
 
-    tsdb.incr(TSDBModel.servicehook_fired, servicehook.id)
+    tsdb.incr(TSDBModel.servicehook_fired, servicehook.id)  # type: ignore
```

Comment:
this is not the correct fix -- this should use `tsdb.backend.incr`

Reply:
please fix the other file too

Reply:
thanks for letting me know! will fix in both files

Reply:
looks like there's a few other files where the same was done (with a typing ignore comment or in a file that is skipped for typing) so i'll look into fixing those as well